src/analysis/seminorm.lean

```diff
@@ -468,16 +609,19 @@ begin
 end
 
 section norm_one_class
-variables [norm_one_class 𝕜] (p : seminorm 𝕜 E)
+variables (p : seminorm 𝕜 E)
```

Comment:
If there is no variable `norm_one_class` the section should also not be called `norm_one_class`.

Reply:
I changed it to `seminorm`.